modules/persistent-storage-vsphere-backup.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-pv-backup_{context}"]
+= Backing up VMware vSphere volumes
+
+{product-title} provisions new volumes as independent persistent disks to freely attach and detach the volume on any node in the cluster. As a consequence, it is not possible to back up volumes that use snapshots. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[Snapshot Limitations] for more information.
```

Comment:
> ...it is not possible to back up volumes that use snapshots.

I'm fine with this as is, considering that it was in the 3.11 docs without complaint or issue; however, I wonder if we should be more explicit:

> it is not possible to back up volumes that use snapshots, or restore volumes from created snapshots.